test/unit/modules/delegates.js

```diff
@@ -756,6 +756,32 @@ describe('delegates', () => {
 						initialSate['1']
 					);
 				});
+
+				it('should not mutate the __private.delegateListCache object when there is only 1 cached round.', () => {
```

Comment:
I thought it should clear the list if there is just one cached round.

Reply:
See: https://github.com/LiskHQ/lisk/pull/2612#issuecomment-444804969